docs/dev/build-instructions/find_build_number.md

```diff
@@ -10,3 +10,14 @@ The build number is automatically increased with every commit that is made to Mu
 https://mumble.info/build-number.json. Simply look up the current commit hash (topmost entry in the result of `git log`) in the list and use the
 corresponding build number.
 
+Instead of manually looking up the commit of interested yourself, you could instead also use the script `scripts/mumble-build-number.py`. You have to
```

Comment:
```suggestion
Instead of manually looking up the commit of interest yourself, you could instead use the script `scripts/mumble-build-number.py`. You have to
```